libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -446,7 +446,7 @@ func (m *Manager) Set(container *configs.Config) error {
 }
 
 func getUnitName(c *configs.Cgroup) string {
-	return fmt.Sprintf("%s-%s.scope", c.Parent, c.Name)
+	return fmt.Sprintf("%s-%s.scope", c.ScopePrefix, c.Name)
```

Comment:
I think we need to clarify some of the behavior in #141. I will wait for some conclusion there :)